uportal-war/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -139,8 +139,8 @@
         </a>
         <c:forEach var="image" items="${thumbnailImages}" varStatus="status">
             <a href="#">
-                <img src="${image}" />
-                <span class="caption">Background ${status.index + 1}</span>
+                <img src="${image}" alt=${imageCaptions[status.index]}/>
+                <span class="caption">${imageCaptions[status.index]}</span>
```

Comment:
> Bigger picture: background images are decoration, right? What's the use case for screen-reader users to select among decorative background images?

Whether there is a use case or not, WCAG auditing tools will flag `<img>` tags without the `alt` attribute.
There should be at least an `alt=""` and preferably a `role="presentation"` attached.

ref: https://www.w3.org/WAI/tutorials/images/decorative/

> Would screen reader users be better served by this feature (both the selection of decorative background images, and the presentation of decorative background images) responsively degrading out of the product when used on a screen reader?

This could be beneficial.
`aria-hidden="true"` would do the trick for most screen readers.
Even with graceful degradation, some `alt` attribute needs to be in place to satisfy WCAG audit tools.